packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -286,7 +286,11 @@ export function validateConfig(config: any): PluginConfigType {
 
   // explicitly block using use_frameworks and Flipper in iOS
   // https://github.com/facebook/flipper/issues/2414
-  if (config?.ios?.flipper !== undefined && config?.ios?.useFrameworks !== undefined) {
+  if (
+    config.ios?.flipper !== undefined &&
+    config.ios?.flipper !== false &&
```

Comment:
Should we check using `!!` instead? That way we would support stuff like `null` or `''`
```suggestion 
    !!config.ios?.flipper !== false &&
```

Reply:
i feel like the double negative makes the statement unclear. looking the `!!config.ios?.flipper !== false` at first glance, it would be hard for me to figure out what the exact value would be. i still prefer to make it clear. any better alternatives?

Reply:
I think it's ok to keep it as is too, or maybe use `Boolean(config.ios?.flipper)`

Reply:
oh `Boolean(config.ios?.flipper)` is neat 👏  like this idea pretty much. thanks, i'll update the pr accordingly.